library/include/internal/hipsolver-compat.h

```diff
@@ -18,6 +18,8 @@
 
 typedef void* hipsolverDnHandle_t;
 
+typedef void* hipsolverGesvdjInfo_t;
```

Comment:
I sort of wondering if we should be using an opaque type rather than `void`. It's too late to consider the change now, I think, but it would ensure `hipsolverDnHandle_t` and `hipsolverGesvdjInfo_t` are distinct types.

I suppose we can always keep that in mind as something to consider doing during a future major version bump.